src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -39,10 +39,10 @@ public class PersonBuilder {
     public PersonBuilder() {
         name = new Name(DEFAULT_NAME);
         phone = new Phone(DEFAULT_PHONE);
-        email = new Email(DEFAULT_EMAIL);
-        roomNumber = new RoomNumber(DEFAULT_ROOMNUMBER);
-        telegram = new Telegram(DEFAULT_TELEGRAM);
-        birthday = new Birthday(DEFAULT_BIRTHDAY);
+        email = null;
+        roomNumber = null;
+        telegram = null;
+        birthday = null;
```

Comment:
May I know why are the fields set to null?

Reply:
Optional attribute are initialised to null as they may not be added when people are created. So it should by default test for null attributes. If non null attributes are require please use the withAttribute methods to do so.